doc/source/f2py/code/meson.build

```diff
@@ -22,9 +22,9 @@ incdir_f2py = run_command(py3,
 
 fibby_source = custom_target('fibbymodule.c',
                             input : ['fib1.f'],
-                            output : ['fibbymodule.c'],
+                            output : ['fibbymodule.c', 'fibby-f2pywrappers.f', 'fibby-f2pywrappers2.f90'],
```

Comment:
Why the `2` in `fibby-f2pywrappers2.f90'`, is that a typo? If it's on purpose, why not drop the `2`?

Reply:
It isn't a typo. The generated file is `blah-f2pywrappers2.f90`. Changing the filename would be a BC-breaking change.

Reply:
Okay thanks. Looks kinda odd, but it is what it is. I think I never noticed because SciPy's Fortran code is pretty much all F77.